Mathlib/Probability/Moments/MGFAnalytic.lean

```diff
@@ -119,6 +122,139 @@ lemma hasFPowerSeriesAt_mgf (hv : v ∈ interior (integrableExpSet X μ)) :
   convert (analyticAt_mgf hv).hasFPowerSeriesAt
   rw [iteratedDeriv_mgf hv]
 
-end Analytic
+lemma differentiableAt_mgf (ht : t ∈ interior (integrableExpSet X μ)) :
+    DifferentiableAt ℝ (mgf X μ) t := (analyticAt_mgf ht).differentiableAt
+
+lemma analyticOnNhd_iterated_deriv_mgf (n : ℕ) :
+    AnalyticOnNhd ℝ (deriv^[n] (mgf X μ)) (interior (integrableExpSet X μ)) :=
```

Comment:
What about using `iteratedDeriv` instead of `deriv^[n]`, here and elsewhere?

Reply:
Ok, I'll change it! I admit that I am still a bit lost in the API for analysis in those PRs about moment generating functions (it also took me some time to understand how the various analytic definitions were related to each other, and which was used when). I only used `deriv^[n]` here instead of `iteratedDeriv` because that's the thing on which I found the result I wanted to use.